src/components/editor/ArticleController.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+
+const categories = [
+  "Category 1",
+  "Category 2",
+  "Category 3"
+];
+
+export default class ArticleController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.handleCategorySubmit = this.handleCategorySubmit.bind(this);
```

Comment:
Yeah, I just did it all because I'm expecting to need it, since I haven't written the handle submit functions yet. If it ends up being redundant I'll remove them.